README.md

```diff
@@ -46,12 +46,20 @@ Citus Cloud runs on top of AWS as a fully managed database as a service and has
 
 If you're looking to get started locally, you can follow the following steps to get up and running.
 
-* Install docker-compose: [Mac][mac_install] | [Linux][linux_install]
-* (Mac only) connect to Docker VM
-  ```bash
-  eval $(docker-machine env default)
-  ```
-
+* Install Docker Engine and docker-compose
+  * Mac:
+    1. Install the [Docker Platform](https://www.docker.com/products/overview#/install_the_platform).
```

Comment:
Yeah, I saw that this morning, poked around to make sure everything was still available for us to use (the HN crowd were _ridiculously_ irresponsibly scaremongering about how this meant Docker "wasn't available" or "wouldn't get new features" or how they were "disappointed", etc.)

I think "Community Edition" scares people because they assume most active development occurs in the closed version and never hops the fence to community, rather than Enterprise features being add-ons. Probably persists from back in the WebSphere days, etc.